app/charts/index.ts

```diff
@@ -947,10 +989,11 @@ const chartConfigsAdjusters: ChartConfigsAdjusters = {
 
           if (tableSegment) {
             newSegment = {
-              ...tableSegment,
+              ...tableSegment.segment,
```

Comment:
Ok, I've seen that the `convertTableFieldsToSegmentField` was in fact modified – I think it would be good to name it in a different way to indicate that we are in fact returning segment and color fields, maybe

```typescript
const {segment, color} = convertTableFieldsToSegmentAndColorFields(...);
```

?

Reply:
However we can't destructure color and segment like that since they can be undefined, I renamed the tableSegment to tableField instead